docs/self-managed/setup/deploy/amazon/amazon-eks/eks-helm.md

```diff
@@ -408,12 +408,8 @@ identity:
 
 Once you've prepared the `values.yml` file, run the following `envsubst` command to substitute the environment variables with their actual values:
 
-```bash
-# generate the final values
-envsubst < values.yml > generated-values.yml
-
-# print the result
-cat generated-values.yml
+```bash reference
+https://github.com/camunda/camunda-deployment-references/blob/8.8/generic/kubernetes/single-region/procedure/assemble-envsubst-values.sh
```

Comment:
for myself to not forget, mentioned this one as `blob/8.8` was no covered by search and replace